config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml

```diff
@@ -46,9 +46,10 @@ periodics:
       - --test
       - --check-version-skew=false
       - --down
+      - --dump=${ARTIFACTS}
       # specific e2e test args
       # TODO(bentheelder): num-nodes in particular is brittle. it would be better for kubetest to handle this
-      - --test_args=--ginkgo.focus=\[Conformance\] --ginkgo.skip=\[Serial\]|Alpha|Kubectl|\[(Disruptive|Feature:[^\]]+|Flaky)\] --disable-log-dump=true --num-nodes=3
+      - --test_args=--report-dir=${ARTIFACTS} --disable-log-dump=true --num-nodes=3 --ginkgo.focus=\[Conformance\] --ginkgo.skip=\[Serial\]|Alpha|Kubectl|\[(Disruptive|Feature:[^\]]+|Flaky)\]
```

Comment:
the first two all jobs will need, --num-nodes will depend on config, the rest of this line will be per-job